src/Dev/Tasks/MigrateFileTask.php

```diff
@@ -21,9 +21,12 @@
 use SilverStripe\Assets\Dev\Tasks\SecureAssetsMigrationHelper;
 use SilverStripe\UserForms\Task\RecoverUploadLocationsHelper;
 use \Bramus\Monolog\Formatter\ColoredLineFormatter;
+use SilverStripe\Dev\Deprecation;
 
 /**
  * Migrates all 3.x file dataobjects to use the new DBFile field.
+ *
+ * @deprected 4.12.0 Will be removed without equivalent functionality to replace it
```

Comment:
Deprecating this class because it uses a load of other deprecated classes from silverstripe/assets.  It was used to migrate CMS 3 files.